### PR TITLE
Cult of the Honkmother - The Clown Sect

### PR DESCRIPTION
**Pull request by Alagoinha**
Special thanks to Theos, Gus, and Redmoogle for helping this code potato

# Wiki Documentation

Rites
Holy Pie - Creates a cream pie  (50 favor)
Honk a Bot - Summons a honkbot (150 favor)
Banana Blessing - Creates a sheet of bananium (1000)

# Changelog
:cl:  
rscadd: Added Honkmother sect
/:cl:

### CODE REVIEW

**Comment by ynot01** [comment]
please write somewhere that it wants bananas

File: code/modules/religion/religion_sects.dm [hunk 1]
```diff
@@ -370,4 +370,51 @@
...
+	
+/**** The Honkmother sect, sacrifice bananas to feed your prank power. ****/
+
+/datum/religion_sect/honkmother
+	name = "The Honkmother"
+	desc = "A sect dedicated to the Honkmother"
+	convert_opener = "The Honkmother welcomes you to her to the party, prankster.<br>Sacrifice bananas to power our pranks and grant you favor."
+	alignment = ALIGNMENT_NEUT
+	max_favor = 10000
+	desired_items = list(/obj/item/reagent_containers/food/snacks/grown/banana)
```
Comment (Hopekz) [inline code/modules/religion/religion_sects.dm]:
```suggestion
	desired_items = list(obj/item/reagent_containers/food/snacks/grown/banana)
```
will fix this error probably

**Review by Hopekz — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
I think that creating a honkbot should be more expensive then 150 flavour

**Comment by JamieD1** [comment]
Please fix compile

**Comment by Alagoinha** [comment]
> Please fix compile

trying to find out how to
at one point it will work

File: code/modules/religion/religion_sects.dm [hunk 2]
```diff
@@ -370,4 +370,51 @@
 	else
 		adjust_favor(75, user)
 	qdel(offering)
+	return 
+	
+/**** The Honkmother sect, sacrifice bananas to feed your prank power. ****/
```
Comment (JamieD1) [inline code/modules/religion/religion_sects.dm]:
```suggestion
/// The Honkmother sect, sacrifice bananas to feed your prank power. 
```

File: code/modules/religion/religion_sects.dm [hunk 3]
```diff
@@ -370,4 +370,51 @@
...
+
+/datum/religion_sect/honkmother
+	name = "The Honkmother"
+	desc = "A sect dedicated to the Honkmother"
+	convert_opener = "The Honkmother welcomes you to her to the party, prankster.<br>Sacrifice bananas to power our pranks and grant you favor."
+	alignment = ALIGNMENT_NEUT
+	max_favor = 10000
+	desired_items = list(obj/item/reagent_containers/food/snacks/grown/banana)
```
Comment (JamieD1) [inline code/modules/religion/religion_sects.dm]:
```suggestion
	desired_items = list(/obj/item/reagent_containers/food/snacks/grown/banana)
```

**Review by JamieD1 — DISMISSED** [review]

File: code/modules/religion/religion_sects.dm [hunk 4]
```diff
@@ -370,4 +370,51 @@
...
+	
+/// The Honkmother sect, sacrifice bananas to feed your prank power. 
+
+/datum/religion_sect/honkmother
+	name = "The Honkmother"
+	desc = "A sect dedicated to the Honkmother"
+	convert_opener = "The Honkmother welcomes you to her to the party, prankster.<br>Sacrifice bananas to power our pranks and grant you favor."
+	alignment = ALIGNMENT_NEUT
+	max_favor = 10000
+	desired_items = list(/obj/item/reagent_containers/food/snacks/grown/banana)
```
Comment (JamieD1) [inline code/modules/religion/religion_sects.dm:383]:
```suggestion
	desired_items = list(/obj/item/reagent_containers/food/snacks/grown/banana)
```

Comment (JamieD1) [inline code/modules/religion/religion_sects.dm]:
```suggestion
```

**Review by Bibby0110 — APPROVED** [review]

**Comment by Alagoinha** [comment]
thanks everyone for the help fixing this mess, it will help me a lot with the other sects I want to try and code

**Pull request merged**